README.rst

```diff
@@ -17,6 +17,11 @@ Clone and install.
 
    git clone https://github.com/NSLS-II/bluesky-browser
    cd bluesky-browser
+   conda create -n bluesky_browser python=3 \
+       bluesky jsonschema matplotlib ophyd pyqt \
+       pyzmq qtpy suitcase-jsonl tornado traitlets  \
+       -c lightsource2-tag
+   conda activate bluesky_browser
```

Comment:
Can these 2 conda commands be moved before these lines?
```bash
   git clone https://github.com/NSLS-II/bluesky-browser
   cd bluesky-browser
```
That will make the consequent installation steps to be in a more logical order.

Reply:
Absolutely since they no longer rely on `requirements.txt`